corehq/apps/users/models.py

```diff
@@ -2548,7 +2550,3 @@ def get(self, user_id):
             user = CouchUser.get_by_user_id(user_id)
             self.cache[user_id] = user
             return user
-
-
-from .signals import *
```

Comment:
it looks like we should replace this with an `AppConfig`. you can see @snopoke's recent change to fluff as an example: https://github.com/dimagi/fluff/pull/105